### ISSUE TITLE
Disabling save checkpoint every N steps causes training to fail

### ISSUE THREAD

**Issue by ShinkoNet** [d8ahazard/sd_dreambooth_extension#16]
For the option Save a checkpoint every N steps, 0 to disable
When setting it to 0, at the end of training, this happens:

```
Traceback (most recent call last):
  File "C:\stable-diffusion-webui\modules\ui.py", line 185, in f
    res = list(func(*args, **kwargs))
  File "C:\stable-diffusion-webui\webui.py", line 54, in f
    res = func(*args, **kwargs)
  File "C:\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\dreambooth.py", line 256, in start_training
    trained_steps = main(config)
  File "C:\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\train_dreambooth.py", line 785, in main
    save_ckpt = not global_step % args.save_embedding_every and global_step != 0
ZeroDivisionError: integer division or modulo by zero
```
And the checkpoint does not get saved, have to restart from scratch.

**Comment by hugostranger** [comment]
Whoops, just duplicated this here: https://github.com/d8ahazard/sd_dreambooth_extension/issues/17

A workaround I'm temporarily using is to set the number higher than the number of training steps so it never reaches the generation point.

**Comment by d8ahazard** [comment]
Closed, duplicate of #17
